src/main/java/org/traccar/database/NotificationManager.java

```diff
@@ -92,9 +92,10 @@ public void updateEvent(Event event, Position position) {
                     if (getById(notificationId).getType().equals(event.getType())) {
                         boolean filter = false;
                         if (event.getType().equals(Event.TYPE_ALARM)) {
-                            String alarms = notification.getString("alarms");
-                            if (alarms == null || !alarms.contains(event.getString(Position.KEY_ALARM))) {
-                                filter = true;
+                            String alarmsStr = notification.getString("alarms");
```

Comment:
Please rename. We don't use shortened words with some rare historical exceptions.

Reply:
done